src/main/java/org/thoughtslive/jenkins/plugins/jira/steps/WaitForIssueUpdateStep.java

```diff
@@ -0,0 +1,148 @@
+package org.thoughtslive.jenkins.plugins.jira.steps;

```

Comment:
You are right but I think it is better to move all steps at once in a dedicated PR.

Reply:
Not sure if I get to remaining steps soon, so I was wondering if we can at least move these new steps to their own folders as this is new type of steps compare to existing ones, just to avoid the confusion.